core/quivr_core/processor/implementations/megaparse_processor.py

```diff
@@ -55,14 +57,35 @@ def processor_metadata(self):
         }
 
     async def process_file_inner(self, file: QuivrFile) -> list[Document]:
-        # mega_parse = MegaParse(file_path=file.path, config=self.megaparse_config)  # type: ignore
-        # document: Document = await mega_parse.aload()
-        # if len(document.page_content) > self.splitter_config.chunk_size:
-        #     docs = self.text_splitter.split_documents([document])
-        #     for doc in docs:
-        #         # if "Production Fonts (maximum)" in doc.page_content:
-        #         #    print('Doc: ', doc.page_content)
-        #         doc.metadata = {"chunk_size": len(self.enc.encode(doc.page_content))}
-        #     return docs
-        # return [document]
-        return []
+        megaparse_url = os.getenv("MEGAPARSE_URL_API")
+        with open(file.path, "rb") as f:
+            files = {"file": (os.path.basename(file.path), f)}
+            response = requests.post(f"{megaparse_url}/file", files=files)
```

Comment:
This call is synchronous. We can use `httpx` to get the same result but asynchronously